ivy.xml

```diff
@@ -0,0 +1,6 @@
+<ivy-module version="2.0">
+    <info organisation="org.github.broadinstitute" module="picard"/>
+    <dependencies>
+        <dependency org="com.github.samtools" name="htsjdk" rev="2.3.0" />
```

Comment:
Are there any non-htsjdk dependencies currently checked in as jars (eg., guava) that could be pulled down from maven as well?